bundles/org.openhab.binding.emby/src/main/java/org/openhab/binding/emby/internal/handler/EmbyBridgeHandler.java

```diff
@@ -0,0 +1,167 @@
+        }
+    }
+
+    public void sendCommand(String commandURL) {
+        try {
+            httputils.doPost(commandURL, "", 2);
+        } catch (EmbyHttpRetryExceeded e) {
+            logger.debug("The number of retry attempts was exceeded", e.getCause());
+        }
+    }
+
+    @Nullable
+    public String sendCommandWithReturn(String commandURL) {
+        try {
+            return httputils.doPost(commandURL, "", 2);
+        } catch (EmbyHttpRetryExceeded e) {
+            logger.debug("The number of retry attempts was exceeded", e.getCause());
+            return null;
+        }
+    }
+
+    private String getServerAddress() {
+        String host = getConfig().get(HOST_PARAMETER).toString();
+        String port = Integer.toString(getIntConfigParameter(WS_PORT_PARAMETER, 8096));
+
+        return host + ":" + port;
+    }
+
+    private int getIntConfigParameter(String key, int defaultValue) {
+        Object obj = this.getConfig().get(key);
+        if (obj instanceof Number) {
+            return ((Number) obj).intValue();
+        } else if (obj instanceof String) {
+            return Integer.parseInt(obj.toString());
+        }
+        return defaultValue;
+    }
+
+    @Override
+    public void initialize() {
+        updateStatus(ThingStatus.UNKNOWN);
+        // Example for background initialization:
+        scheduler.execute(() -> {
+            try {
+                String host = getConfig().get(HOST_PARAMETER).toString();
+                if (host == null || host.isEmpty()) {
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                            "No network address specified");
+                } else {
+                    connection.connect(host, getIntConfigParameter(WS_PORT_PARAMETER, 8096),
+                            (String) this.getConfig().get(API_KEY), scheduler,
+                            getIntConfigParameter(REFRESH_PARAMETER, 10000),
+                            getIntConfigParameter(WS_BUFFER_SIZE, 100000));
+
+                    connectionCheckerFuture = scheduler.scheduleWithFixedDelay(() -> {
+                        if (!(connection.checkConnection())) {
+                            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                                    "Connection could not be established");
+                        }
+                    }, 1, getIntConfigParameter(REFRESH_PARAMETER, 10), TimeUnit.SECONDS);
+                }
+            } catch (Exception e) {
+                logger.debug("error during opening connection: {}", e.getMessage(), e);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getLocalizedMessage());
+            }
+        });
+    }
+
+    @Override
+    public void handleEvent(EmbyPlayStateModel playstate, String hostname, int embyport) {
+        logger.debug("Received event from emby server");
```

Comment:
yes this message can and should be removed. If we successfully receive and event we will log that to the debug logging level.  and if it is null it will also be logged.